packages/ckeditor5-cloud-services/src/cloudservices.js

```diff
@@ -38,6 +39,14 @@ export default class CloudServices extends ContextPlugin {
 			this[ optionName ] = options[ optionName ];
 		}
 
+		/**
+		 * Map of `Token` object instances keyed by `tokenUrl`s.
+		 *
+		 * @private
+		 * @type {Map.<String, module:cloud-services-core/token~Token>}
```

Comment:
This accepts a string or a function as the key (which is actually a bit odd :smile:)

Reply:
Unfortunately, then it displays in docs like this: 

```
Map.&lt(String|Function), module:cloud-services-core/token~Token>
```

Reply:
but with current notation it's:

Reply:
```
Map.<String, Token>
```

and `Token` is linking

Reply:
Ok. Let's go with this option then.